src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splitString = sourceString
+    .split(';')
+    .filter((string) => string.includes(':'));
+  const result = {};
+
+  const callback = (line) => {
+    if (line) {
+      const splitLine = line.split(':');
+
+      result[splitLine[0].trim()] = splitLine[1].trim();
+    }
+  };
```

Comment:
[CODE KNOWLEDGE] - You don't need to check for the existence of `line` in the `callback` function since the `.filter` method has already ensured that each element includes a colon, which implies that the line is not empty. You can remove the `if (line)` check.